src/network/network_gamelist.cpp

```diff
@@ -179,6 +179,8 @@ void NetworkGameListRequery()
 				NetworkUDPQueryServer(static_cast<ServerAddressDirect *>(item->address.get())->address);
 				break;
 
+			/* Will never support SERVER_ADDRESS_JOIN_KEY. */
```

Comment:
I would make it `case SERVER_ADDRESS_JOIN_KEY: // Will never be supported` so it becomes slightly more apparent in the IDE instead of just a comment.

Reply:
It was more a mental note, as that function gets removed a few commits later :)